README.md

```diff
@@ -1,8 +1,151 @@
-# sindri-scroll-sdk
+# Sindri Provers for Scroll SDK
 
-## Temporary Usage Instructions
 
-First, create your `config.json` file from a template
+<img src="./media/sindri-gradient-logo.webp" height="160" align="right"/>
+
+#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/) | [Getting Started](#getting-started) | [Development](#development)
+
+Sindri provides automated ZK proving infrastructure, empowering hundreds of teams — including leading Layer 2s and rollups — to launch in minutes instead of months.
+Through our API, developers can seamlessly integrate verifiable computation, reducing time to market, cutting costs, and scaling faster.
+Sindri makes zero-knowledge infrastructure simple and accessible, facilitating automation across every layer of the ZK app deployment stack.
+
+This repository hosts the images and Helm charts designed to perform proof generation for a Scroll SDK chain.
+The following diagram depicts how the Scroll SDK components work together when you use Sindri as a prover.
+```mermaid
+---
+title: Scroll SDK with Sindri Provers
+---
+flowchart LR
+    subgraph settle[Settlement Layer]
+        direction LR
+    end
+    subgraph sequence[Sequencing Layer]
+        direction LR
+    end
+    subgraph prove[Proving Layer]
+        direction LR
+        C["**Coordinator**"] -.->|Tasks| P@{ shape: processes, label: "**Provers**\nThese components outsource proof generation to Sindri's GPU workers" }
+        P -.->|Proofs| C
+    end
+    subgraph sindri["Scroll Circuits on Sindri"]
+        direction TB
+        Chunk["**Chunk**
+        Verified execution of contiguous blocks (i.e. a zkEVM proof)"]
+        Batch["**Batch**
+        Intermediate aggregation of multiple chunks"]
+        Bundle["**Bundle**
+        Final aggregate proof sent to the settlement layer"]
+    end
+    settle o--o sequence
+    sequence o--o prove
+    P --> Chunk
+    P --> Batch
+    P --> Bundle
+```
+
+For more information about the Sindri platform, please check out [sindri.app](https://sindri.app/).
+The next section will show you how to launch a local Scroll SDK devnet with provers running on Sindri.
```

Comment:
```suggestion
The next section shows how to launch a local Scroll SDK devnet with provers running on Sindri.
```

Reply:
I'd also like to see a title and link to the "next section" just in case text moves around and for absolute clarity. So 

```suggestion
The next section ([Getting Started](#getting-started)) shows how to launch a local Scroll SDK devote with provers running on Sindri.
```